kbi/0022/index.rst

```diff
@@ -0,0 +1,103 @@
+.. index::
+   single: datalad save; git mv
+
+KBI0022: Performance of ``mv`` vs ``git mv`` when renaming dataset directories 
+==============================================================================
+
+:authors: Stephan Heunis <jsheunis@gmail.com>
+:discussion: <link>
+:keywords: performance, time, datalad save, mv, git move, rename
+:software-versions: datalad_0.18.4, git-annex_10.20230330-g98a3ba0ea
+
+When renaming a directory that forms part of many files in a large DataLad dataset,
```

Comment:
I'm not sure I parse this sentence from "directory that forms part of many files". From reading #74 it sounds like the following phrase would capture it, too? (but I wasn't around when the discussion happened, so I might misread) 

```suggestion
When renaming a directory that contains many files in DataLad dataset,
```

Reply:
No, your suggestion works better 👍